README.md

```diff
@@ -8,44 +8,45 @@ A CLI and scripts used at Echobind
 [![License](https://img.shields.io/npm/l/eb-scripts.svg)](https://github.com/echobind/eb-scripts/blob/master/package.json)
 
 <!-- toc -->
-
-- [eb-scripts](#eb-scripts)
-- [Usage](#usage)
-- [Commands](#commands)
-  <!-- tocstop -->
+* [eb-scripts](#eb-scripts)
+* [Usage](#usage)
+* [Commands](#commands)
+* [Contributing](#contributing)
+<!-- tocstop -->
 
 # Usage
 
 <!-- usage -->
-
 ```sh-session
 $ npm install -g eb-scripts
 $ eb-scripts COMMAND
 running command...
 $ eb-scripts (-v|--version|version)
-eb-scripts/0.0.0 darwin-x64 node-v10.16.0
+eb-scripts/0.0.0-development darwin-x64 node-v10.16.0
```

Comment:
this seems off? should development be here?

Reply:
I'm not sure why that got added by `oclif`...

It's weird because the way `semantic-release` works, is it creates a tag on `git`, publishes to npm, but then it doesn't update the `package.json` in `master`?

Any thoughts on how to get around this? This discussion can happen after this is merged in though.

Reply:
Filed an issue to take care of this #10